src/graphql/issue.rs

```diff
@@ -3,18 +3,96 @@ use std::fmt;
 use anyhow::Context as AnyhowContext;
 use async_graphql::{
     connection::{query, Connection, EmptyFields},
-    Context, Object, Result, SimpleObject,
+    scalar, Context, Object, Result, SimpleObject,
 };
+use serde::{Deserialize, Serialize};
 
 use crate::database::{self, Database, TryFromKeyValue};
+use crate::github::IssueState;
+use crate::github::PullRequestState;
 
-#[derive(SimpleObject)]
+scalar!(IssueState);
+scalar!(PullRequestState);
+
+#[derive(SimpleObject, Serialize, Deserialize)]
 pub(crate) struct Issue {
+    pub(crate) id: String,
     pub(crate) owner: String,
     pub(crate) repo: String,
-    pub(crate) number: i32,
+    pub(crate) number: i64,
```

Comment:
The original `number` field was defined as an `i32`, which led to repeated and unnecessary conversions from `i64` to `i32`. To address this, I've updated the field type. I'd appreciate your thoughts on this change.